buddyboss/buddyboss-platform
# Activity post form Gif icon highlighted issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1197  .

### How to test the changes in this Pull Request:

1. Go to the activity page
2. Click on the Gif icon
3. Click outside anywhere
4. See the error is not happening

### Proof Screenshots or Video

https://www.loom.com/share/124309e68c0147a7b0419789ca00b753

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
